app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -108,8 +108,7 @@ def upload
 
       # repository url
       if (repo_url = params[:repository_url])
-        @provider.repository_url = repo_url
-        @provider.save!
+        @organization.cdn_configuration.update!(url: repo_url)
```

Comment:
@parthaa @jlsherrill I wonder if we should remove the repository_url param from this API as part of this change? My thinking is that this code doesn't run the CdnConfiguration::Update (previously Provider::Update) action when making the change - so I don't see its purpose. Notice that the organizations controller does run the dynflow action - and that's what the existing UI uses.

Reply:
The API param doesn't show up currently in hammer or foreman-documentation repo so it should be safe

Reply:
i think it would be fine to remove it entirely, since its broken as is